scout/quickstart.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Docker Scout quickstart
+keywords: scout, supply chain, vulnerabilities, packages, cves, scan, analysis, analyze
+description: >
+  Learn how to get started with Docker Scout to analyze images and fix vulnerabilities
+redirect_from:
+  - /atomist/get-started/
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout analyzes image contents and generates a detailed report of packages
+and vulnerabilities that it detects. Docker Scout can provide you with
+suggestions for how to remediate issues discovered by image analysis.
+
+The guide takes a vulnerable application image and shows you how to use Docker
+Scout to identify and fix those vulnerabilities, compare changes over time,
+and share results with your team.
+
+You can use [this example project](https://github.com/docker/scout-demo-service)
+to follow along by cloning its repository:
+
+```shell
+git clone https://github.com/docker/scout-demo-service.git
+```
+
+Move into the directory:
+
+```shell
+cd scout-demo-service
+```
+
+Build the image using the intended repository name, tagging it as "v1":
```

Comment:
Intended repository name? Also "tag" refers the full name: `<org-name>/scout-demo:v1`